getting-started/repl/intergration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Integration
+page_title: Telerik REPL for Blazor Integration
+description: Telerik REPL for Blazor integration in other Telerik Resources, Visual Studio and Visual Studio Code
+slug: blazor-repl-integration
+tags: telerik,blazor,repl,integration
+published: True
+position: 10
+---
+
+# Telerik REPL for Blazor Integration
+
+Telerik REPL for Blazor is integrated in our resources (documentation and demos), as well as Visual Studio and Visual Studio Code.
+
+In this article:
+
+* [Telerik UI for Blazor documentation Integration](#telerik-ui-for-blazor-documentation-integration)
```

Comment:
IMHO we don't need to burden the anchor links with "Telerik UI for Blazor". Make the links and headings shorter for easier readability and scanning. (Documentation Integration, Demos Integration, etc.)